app/src/main/java/de/danoeh/antennapod/view/PagerIndicatorView.java

```diff
@@ -50,6 +54,12 @@ private void setup() {
         a.recycle();
     }
 
+    /* Visual and logical position distinction only happens in RTL locales (e.g. Persian)
+     * where pages positions are flipped thus it does nothing in LTR locales (e.g. English) */
+    float logicalPositionToVisual(float position) {
```

Comment:
I think this method can be `private`. Also, please use Javadoc (`/** ... */`) instead of a normal comment, so IDEs can show the comment when looking at callers.

Apart from that, looks good :+1:

Reply:
👍 Done